### DISCUSSION TITLE
Warning about new node or edge type

### DISCUSSION THREAD

**Discussion by psyycker** [xyflow/xyflow#4247] [Q&A]
Hi there!

I'm using this example as a base: https://codesandbox.io/p/sandbox/competent-moon-dpj59k?file=%2FSidebar.js%3A24%2C1&utm_medium=sandpack

But then I'm creating my node following the way it's done in the guides:
```ts
export const nodeTypes = {
  [LOGIC_BLOCK]: LogicBlock,
  [CONDITION_BLOCK]: ConditionBlock,
};
```
(This is outside a component, so completely memoized)

And finally adding them to the React Flow component:
```ts
nodeTypes={nodeTypes}
```
I keep having this warning: `It looks like you've created a new nodeTypes or edgeTypes object.`

I don't have any edge at the moment (I'm just experimenting) so not sure what I'm doing wrong here.

Any idea?

**Comment by moklick** [comment]
This is weird 🤔 What does your dev setup look like?

**Reply by psyycker** [reply]
It's a simple NextJS project. Nothing fancy

**Comment by FallingEpiphany** [comment]
I have this problem even in default settings, without any custom , in nextjs.
